frontend/app/controllers/spree/checkout_controller.rb

```diff
@@ -170,6 +171,21 @@ def before_payment
       end
     end
 
+    def add_store_credit_payments
```

Comment:
Assignment Branch Condition size for add_store_credit_payments is too high. [16.28/15]